From 5d1782e6331406b75e1d4d3eca8fd1aacc9fb190 Mon Sep 17 00:00:00 2001
From: Felix Kuehling <Felix.Kuehling@amd.com>
Date: Mon, 23 Apr 2018 21:16:21 -0400
Subject: [PATCH 4277/5725] drm/amdkfd: Align Makefile with upstream

Changed includes of amd_rdma.h to work with the upstream include paths
and removed unnecessary inclusion in kfd_priv.h.

Change-Id: Id624bf0e358bd416348986f756be7b51e242fc3c
Signed-off-by: Felix Kuehling <Felix.Kuehling@amd.com>
Signed-off-by: Kalyan Alle <kalyan.alle@amd.com>
---
 drivers/gpu/drm/amd/amdkfd/Makefile         | 7 ++-----
 drivers/gpu/drm/amd/amdkfd/kfd_peerdirect.c | 2 +-
 drivers/gpu/drm/amd/amdkfd/kfd_priv.h       | 1 -
 drivers/gpu/drm/amd/amdkfd/kfd_rdma.c       | 2 +-
 4 files changed, 4 insertions(+), 8 deletions(-)

diff --git a/drivers/gpu/drm/amd/amdkfd/Makefile b/drivers/gpu/drm/amd/amdkfd/Makefile
index b65537a..66f1921 100644
--- a/drivers/gpu/drm/amd/amdkfd/Makefile
+++ b/drivers/gpu/drm/amd/amdkfd/Makefile
@@ -23,11 +23,8 @@
 # Makefile for Heterogenous System Architecture support for AMD GPU devices
 #
 
-FULL_AMD_PATH=$(src)/..
-
-ccflags-y := -Iinclude/drm \
-		-I$(FULL_AMD_PATH)/include/  \
-		-I$(FULL_AMD_PATH)/include/asic_reg
+ccflags-y := -Idrivers/gpu/drm/amd/include/  \
+		-Idrivers/gpu/drm/amd/include/asic_reg
 
 amdkfd-y	:= kfd_module.o kfd_device.o kfd_chardev.o kfd_topology.o \
 		kfd_pasid.o kfd_doorbell.o kfd_flat_memory.o \
diff --git a/drivers/gpu/drm/amd/amdkfd/kfd_peerdirect.c b/drivers/gpu/drm/amd/amdkfd/kfd_peerdirect.c
index 1b1a0ca..87344cc 100644
--- a/drivers/gpu/drm/amd/amdkfd/kfd_peerdirect.c
+++ b/drivers/gpu/drm/amd/amdkfd/kfd_peerdirect.c
@@ -49,9 +49,9 @@
 #include <linux/slab.h>
 #include <linux/scatterlist.h>
 #include <linux/module.h>
+#include <drm/amd_rdma.h>
 
 #include "kfd_priv.h"
-#include "amd_rdma.h"
 
 
 
diff --git a/drivers/gpu/drm/amd/amdkfd/kfd_priv.h b/drivers/gpu/drm/amd/amdkfd/kfd_priv.h
index 7cd4819..25bfc4f 100644
--- a/drivers/gpu/drm/amd/amdkfd/kfd_priv.h
+++ b/drivers/gpu/drm/amd/amdkfd/kfd_priv.h
@@ -39,7 +39,6 @@
 #include <linux/kfifo.h>
 #include <kgd_kfd_interface.h>
 
-#include "amd_rdma.h"
 #include "amd_shared.h"
 
 #define KFD_SYSFS_FILE_MODE 0444
diff --git a/drivers/gpu/drm/amd/amdkfd/kfd_rdma.c b/drivers/gpu/drm/amd/amdkfd/kfd_rdma.c
index 985855f..3454514 100644
--- a/drivers/gpu/drm/amd/amdkfd/kfd_rdma.c
+++ b/drivers/gpu/drm/amd/amdkfd/kfd_rdma.c
@@ -25,7 +25,7 @@
 #include <linux/pid.h>
 #include <linux/err.h>
 #include <linux/slab.h>
-#include "amd_rdma.h"
+#include <drm/amd_rdma.h>
 #include "kfd_priv.h"
 
 
-- 
2.7.4

